Lombiq.Tests.UI/Extensions/ElementRetrievalUITestContextExtensions.cs

```diff
@@ -146,5 +146,5 @@ public static void VerifyElementTexts(this UITestContext context, By by, params
         new(
             context.Driver,
             context.Configuration.TimeoutConfiguration.RetryTimeout,
-            context.Configuration.TimeoutConfiguration.RetryTimeout);
+            context.Configuration.TimeoutConfiguration.RetryInterval);
```

Comment:
This hilarious bug has been with us since the very beginning of the project, and @YevgeniyShunevych pointed it out.